Makefile

```diff
@@ -1,7 +1,7 @@
 SRC := .
 
 env:
-	mkvirtualenv --python=$(which python3.7) up42-py
+	mkvirtualenv --python=$(which python3) up42-py
```

Comment:
Are we sure the python sdk works with all python 3 versions?

Reply:
>3.6, but seems there is no way to use `which` like that. Any idea @marooned20? Otherwise will just use `which python3.8` to indicate the newest version.

Reply:
if the prerequisite to run this is `>= python3.6` then it's better to use a pinned version of python (either of 3.6, 3.7 or 3.8) instead of just python3.